spec/support/api/helpers.rb

```diff
@@ -224,6 +224,12 @@ def expect_options_results(type, data = {})
           expect(response.headers['Access-Control-Allow-Methods']).to include('OPTIONS')
         end
 
+        def expect_unauthorized_request
```

Comment:
minor, but this might read better as expect_forbidden_request, since the status is `:forbidden`